### ISSUE TITLE
JOSS review - License

### ISSUE THREAD

**Issue by craig-willis** [terminological/dtrackr#10]
Review issue: https://github.com/openjournals/joss-reviews/issues/4707
Branch reviewed: release-0.2.4

**Problem**
JOSS requires that your repository include the full-text of an OSI approved license. See the [Software License](https://joss.readthedocs.io/en/latest/review_criteria.html#software-license) section of the review criteria. The current [LICENSE](https://github.com/terminological/dtrackr/blob/release-0.2.4/LICENSE) file does not appear to be an OSI approved license.  The [DESCRIPTION](https://github.com/terminological/dtrackr/blob/b79ee31fe4effd2c40c2d634a237f16fc0440caf/DESCRIPTION#L14) indicates an MIT license. 

If the MIT license has been selected, consider updating the LICENSE file based on the [OSI template](https://opensource.org/licenses/MIT). (Also, I'm not sure what "MIT + file LICENSE" means in the DESCRIPTION).

**Comment by robchallen** [comment]
Many thanks for this and other comments which are all very helpful.

I went through a few iterations with the LICENSE files. CRAN have some specific ways they want the files which I now sort of understand, but required me taking out the original. I've added in a LICENCE.md file into this repo for github, which is ignored for CRAN submission. The 2 line LICENSE text file and DESCRIPTION entry "MIT + file LICENSE" is the correct incantation for CRAN.

**Comment by craig-willis** [comment]
Ah, that makes sense now. I wasn't aware of the CRAN license requirements but see now that this is a common approach. Did you push your change to Github?  I don't see a new commit.

**Comment by robchallen** [comment]
Not yet. 
I'll try and bundle a few things together, so it doesn't come at you piecemeal. 
It will probably will take me a few days.
Do you close these issues or do I?

**Comment by craig-willis** [comment]
Sounds good.

As for closing the issues, I've confirmed with the editor that either of is can close an issue after it's been addressed. I'm happy to review any changes (or explanation of why changes don't need to be made) and close them. You're also welcome to close them and I'll let you know if I have any further concerns.  

For JOSS, the main thing is that I communicate any concerns -- particularly that might block acceptance -- via the primary review issue. I don't plan on putting anything there until you've had a chance to respond to the open issues in this repo.

**Comment by robchallen** [comment]
Hi. I've worked through all your helpful comments. In the end I made quite a few changes which I will describe in the linked issues, and this has been very worthwhile. I've bundled all the changes into a new release (0.2.4.9000) in a new branch which can be found here:

https://github.com/terminological/dtrackr/tree/joss-fixes-0.2.4.9000

On this specific issue I've [added a LICENCE.md file.](https://github.com/terminological/dtrackr/compare/0.2.4...0.2.4.9000#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1d)

 This now show up on the home page of GitHub also.
 
 (p.s. I'll let you close the issues once reviewed)
